libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -814,7 +814,15 @@ def write_UART_config(f):
             rts_line = 'PAL_LINE(GPIO%s,%uU)' % (p.port, p.pin)
         else:
             rts_line = "0"
-        if dev.startswith('OTG'):
+        if dev.startswith('OTG1'):
```

Comment:
first and 3rd branches in if() are the same, why not use two? first check for OTG2, then check for just startswith('OTG')